mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/forms/DraftFormsListFragment.kt

```diff
@@ -0,0 +1,167 @@
+package rs.readahead.washington.mobile.views.fragment.forms
+
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.hzontal.shared_ui.bottomsheet.BottomSheetUtils
+import org.hzontal.shared_ui.bottomsheet.BottomSheetUtils.ActionSeleceted
+import org.hzontal.shared_ui.utils.DialogUtils
+import rs.readahead.washington.mobile.MyApplication
+import rs.readahead.washington.mobile.R
+import rs.readahead.washington.mobile.bus.event.ShowFormInstanceEntryEvent
+import rs.readahead.washington.mobile.databinding.FragmentDraftFormsListBinding
+import rs.readahead.washington.mobile.domain.entity.collect.CollectFormInstance
+import rs.readahead.washington.mobile.mvp.contract.ICollectFormInstanceListPresenterContract
+import rs.readahead.washington.mobile.mvp.presenter.CollectFormInstanceListPresenter
+import rs.readahead.washington.mobile.views.adapters.CollectDraftFormInstanceRecycleViewAdapter
+import rs.readahead.washington.mobile.views.base_ui.BaseBindingFragment
+import rs.readahead.washington.mobile.views.interfaces.ISavedFormsInterface
+import timber.log.Timber
+
+class DraftFormsListFragment : BaseBindingFragment<FragmentDraftFormsListBinding>(
+    FragmentDraftFormsListBinding::inflate
+),
+    FormListInterfce, ICollectFormInstanceListPresenterContract.IView, ISavedFormsInterface {
+    var recyclerView: RecyclerView? = null
```

Comment:
This attribute should be private and late init var because it's going to be initialize it in the onCreateView change it to `    private lateinit var recyclerView: RecyclerView
`or access it directly through the binding `binding?.draftFormInstances`